debarshi/js/components/todoBase.js

```diff
@@ -0,0 +1,55 @@
+import todos from "../constant.js"
+import render from "../todo.js"
+import { todoWrapper } from "../selectors.js"
+
+const createTodoBase = (todo, props = {}) => {
+  const { draggable = false } = props
+
+  const onTodoDragStart = (event) => {
+    const dataId = event.currentTarget.getAttribute("data-id")
+    event.dataTransfer.dropEffect = "move"
+    event.dataTransfer.setData("data_id", dataId)
```

Comment:
two different variable with same name but different convention would be confusing - data-id and data_id